packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -181,10 +181,12 @@ export default function VisualEditor( { styles } ) {
 		}
 
 		if ( themeSupportsLayout ) {
-			return defaultLayout;
+			// We need to ensure support for wide and full alignments,
+			// so we add the column type.
+			return { ...defaultLayout, type: 'column' };
 		}
-
-		return undefined;
+		// Set column layout for classic themes so all alignments are supported.
+		return { type: 'column' };
```

Comment:
We need this type set explicitly here so that the top-level blocks in the post editor always get full/wide alignment controls. 

Ideally, for block themes those controls would only appear if PostContent block had content width, but that's not how it currently works. I opened #42911 to track that issue.

Reply:
On line 248, we pass in the `defaultLayout` object to the `LayoutStyle` component, which is responsible for outputting the default layout rules for the root container and post title wrapper. It looks like we might need to add or pass in the `type: 'column'` here, too?

In testing it appears that with this PR the alignment rules aren't being applied to the post title wrapper (I'm testing with emptytheme):

| Trunk | This PR |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/183344273-2b9efc62-d90e-4297-a635-c5be380c33f9.png) | ![image](https://user-images.githubusercontent.com/14988353/183344285-4c39f757-32e7-4f57-8139-844524f9c896.png) |